app/src/main/java/org/sagebase/crf/CrfActivitiesFragment.java

```diff
@@ -283,7 +306,7 @@ public void scheduleSelected(SchedulesAndTasksModel.ScheduleModel schedule) {
         } else {
             // transition to clinic detail screen
             mClinicDate = schedule.scheduledOn;
-            showClinicActivities();
+            showActivitiesForSchedule();
```

Comment:
I may be missing the complexity, but this seems like a lot of code changes for what I thought was going to be really simple.  It may be a fine solution, but I can't tell if this alters any of our Journey UI states without pulling down the branch.  Can you give me read access to your fork?

Reply:
Yea this was a bit more code than I thought. It'd be great to have you run through it.

You should have read access since it's a public fork and I didn't see any additional permissions. Let me know if that's not the case.

Reply:
Oh yeah, totally.  I was trying to set up your BP fork and not your CRF fork.